js/fullPicture.js

```diff
@@ -0,0 +1,65 @@
+//как мне импортировать не массив а функцию?
+import { generateArrayPictures } from './data.js';
+import { renderCommentsList } from './comments.js';
+
+const bigPicture = document.querySelector('.big-picture');
+const closeButton = document.querySelector('.big-picture__cancel');
+
+/*
+ * функция для проверки id миниатюры и полноразмерной фотографии
+ * @param (string) id
+ */
+function openFullSizePicture (photoId) {
+  const selectedPhoto = generateArrayPictures.find ((photo) => photo.id === photoId);
+  if (selectedPhoto) {
+    renderBigPicture(selectedPhoto);
+    closeButton.addEventListener('click', () => {
+      pushCloseButton();
+    });
+  }
+}
+
+/*
+ * функция закрытия фотографии по нажатию квиши esc
+ */
+
+function pushEscButton (evt) {
+  if (evt.key === 'Escape') {
+    evt.preventDefault();
+    pushCloseButton();
+  }
+}
+
+/*
+ * функция закрытия полноразмерной фотографии по клику на кнопку
+ */
+
+function pushCloseButton () {
+  bigPicture.classList.add('hidden');
+  //обработчик не удаляется
+  document.removeEventListener('keydown', pushEscButton());
```

Comment:
обработчик не удаляется из за того, что передается результат функции, а не ссылка на нее
```suggestion
  document.removeEventListener('keydown', pushEscButton);
```